app/javascripts/deploy_my_codes/controllers/add_project_controller.js

```diff
@@ -0,0 +1,40 @@
+require.register('deploy_my_codes/controllers/add_project_controller', function(exports, require, module){
+  module.exports = function($scope, OrganizationService, ProjectService, UserService) {
+    var bindOrganizations = function(organizations) {
+      $scope.organizations = organizations;
+    };
+
+    var bindUser = function(user) {
+      $scope.user = user;
+    };
+
+    $scope.projects = [];
+    $scope.selectContext = function(context) {
+      _.each($scope.organizations, function(organization) { organization.selected = false });
+      $scope.projects      = [];
+      $scope.user.selected = false;
+      context.selected     = true;
+
+      if (context.isLoggedIn === true) {
+        loadUserProjects();
```

Comment:
Nope only it the `context` respond to `isLoggedIn`. It's the only way I've found to know if the `context` is a  `user` or an `organization`